public/js/add_more_illustrations.js

```diff
@@ -31,11 +29,15 @@ const newItem = (e) => {
     // remove the whole div when clicking the remove btn
     item.querySelector(".btn-remove").addEventListener("click", () => item.remove());
     collectionHolder.appendChild(item);
-    item.querySelector("input").addEventListener("change", function() {previewFile(item)});
+    item.querySelector('input[type=file]').addEventListener("change", function() {previewFile(item)});
     // increase index
     collectionHolder.dataset.index++;
 };
 
 document
     .querySelectorAll('.btn-new')
-    .forEach(btn => btn.addEventListener("click", newItem));
\ No newline at end of file
+    .forEach(btn => btn.addEventListener("click", newItem));
+
+    document
+    .querySelectorAll('.btn-remove')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/F-Jean/SnowTricks/pullRequest?prid=8521799)
```suggestion
    .querySelectorAll(".btn-remove")
```